install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -6,42 +6,82 @@ schema: |-
   definition user {}
 
   definition organization {
+    // Organization specific roles
     relation owner: user
     relation member: user
 
+    // Membership is the union of all organization specific roles
+    permission membership = owner + member
+
+    // Operations on organizations
     permission organization_read = owner + member
     permission organization_write = owner
 
+    // // Operations on Organization's Members
     permission organization_members_read = owner + member
     permission organization_members_write = owner + member
```

Comment:
I don't think we need the `organization_` prefix or should have this.

Reply:
Was intended to distinguish for example `group_members_read` etc

Reply:
but whenever you access this permission you need to qualify it anyway, no?